A4_2.java

```diff
@@ -0,0 +1,55 @@
+import java.util.Scanner;
+
+public class A4_2 {
+	
+public static final int CONSTANT_NUMBER_SEVEN = 7;
+	
+
+	public static void main(String[] args) {
+		// TODO Auto-generated method stub
+
+		
+		 
+		 
+		Scanner scan = new Scanner(System.in);
+		System.out.print("How many numbers will be entered? ");
+
+		int numCount = scan.nextInt();
+		int firstPos = 0;
+		int lastPos = 0;
+		boolean seven = true;
+		
+		
+		
+		
+			for (int n =0 ; n < numCount ; n++ ) {
+		       System.out.print("Eneter the num: ");
+		       int num = scan.nextInt();
+		       if(num==CONSTANT_NUMBER_SEVEN) {
+		    	   if(firstPos == 0) {
+		    		   firstPos = n+1;
+		    		   
+		    	   }
+		    	    
+		    		   lastPos = n;
+		    		   seven = false;
+		    	   }
+}
+		           
+		       
+		    	 if (seven) {

```

Comment:
nit: that work but I would rename the variable to noSeven instead to make it easier to understand